scripts/src/java/org/oppia/android/scripts/proto/maven_dependencies.proto

```diff
@@ -0,0 +1,95 @@
+syntax = "proto3";
+
+package proto;
+
+option java_package = "org.oppia.android.scripts.proto";
+option java_multiple_files = true;
+
+// Represents the list of all third-party maven dependencies.
+message MavenDependencyList {
+  // Stores the list of all third-party maven dependencies.
+  repeated MavenDependency maven_dependency = 1;
+}
+
+// Stores all the information about a third-party maven dependency.
+message MavenDependency {
+  // Name of the third-party dependency.
+  string artifact_name = 2;
+
+  // Version of the third-party dependency.
+  string artifact_version = 3;
+
+  // List of licenses corresponding to a third-party dependency.
+  repeated License license = 4;
+
+  // Represents the origin of the source of the license links.
+  OriginOfLicenses origin_of_license = 5;
```

Comment:
@BenHenning I am a little confused about this field, this isn't used anywhere. Do we really need this field?

Reply:
It was needed before because I was assuming that in some cases the script can correctly infer when something can be scraped, and before you had both primary and alternative links. It may still be a cleaner way to represent the dependencies, though. The origin could always be the original list of license links, and the actual license list could indicate the ones that a human has manually specified to ensure the dependency is correct.

@rt4914 any thoughts?

Reply:
As per conversation, let's go ahead and apply this. It will help us in keeping reference to original links.

Reply:
Made required changes.